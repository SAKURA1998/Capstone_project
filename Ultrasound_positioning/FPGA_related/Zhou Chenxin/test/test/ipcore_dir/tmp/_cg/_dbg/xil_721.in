SET_FLAG DEBUG FALSE
SET_FLAG MODE INTERACTIVE
SET_FLAG STANDALONE_MODE FALSE
SET_PREFERENCE devicefamily virtex5
SET_PREFERENCE device xc5vlx50t
SET_PREFERENCE speedgrade -2
SET_PREFERENCE package ff1136
SET_PREFERENCE verilogsim true
SET_PREFERENCE vhdlsim false
SET_PREFERENCE simulationfiles Structural
SET_PREFERENCE busformat BusFormatAngleBracketNotRipped
SET_PREFERENCE outputdirectory C:/Users/SJTU/Desktop/test/test/ipcore_dir/
SET_PREFERENCE workingdirectory C:/Users/SJTU/Desktop/test/test/ipcore_dir/tmp/
SET_PREFERENCE subworkingdirectory C:/Users/SJTU/Desktop/test/test/ipcore_dir/tmp/_cg/
SET_PREFERENCE transientdirectory C:/Users/SJTU/Desktop/test/test/ipcore_dir/tmp/_cg/_dbg/
SET_PREFERENCE designentry Verilog
SET_PREFERENCE flowvendor Other
SET_PREFERENCE addpads false
SET_PREFERENCE projectname coregen
SET_PREFERENCE formalverification false
SET_PREFERENCE asysymbol false
SET_PREFERENCE implementationfiletype Ngc
SET_PREFERENCE foundationsym false
SET_PREFERENCE createndf false
SET_PREFERENCE removerpms false
SET_PARAMETER Component_Name sin
SET_PARAMETER Functional_Selection Sin_and_Cos
SET_PARAMETER Architectural_Configuration Parallel
SET_PARAMETER Pipelining_Mode No_Pipelining
SET_PARAMETER Data_Format SignedFraction
SET_PARAMETER Phase_Format Scaled_Radians
SET_PARAMETER Input_Width 16
SET_PARAMETER Register_Inputs true
SET_PARAMETER Output_Width 16
SET_PARAMETER Register_Outputs true
SET_PARAMETER Round_Mode Nearest_Even
SET_PARAMETER Iterations 0
SET_PARAMETER Precision 0
SET_PARAMETER Coarse_Rotation true
SET_PARAMETER Compensation_Scaling No_Scale_Compensation
SET_PARAMETER CE false
SET_PARAMETER SCLR false
SET_PARAMETER ND false
SET_PARAMETER RDY false
SET_PARAMETER X_OUT true
SET_PARAMETER Y_OUT false
SET_PARAMETER Phase_Output false
SET_CORE_NAME CORDIC
SET_CORE_VERSION 4.0
SET_CORE_VLNV xilinx.com:ip:cordic:4.0
SET_CORE_CLASS com.xilinx.ip.cordic_v4_0.cordic_v4_0
SET_CORE_PATH D:/Xilinx/14.6/ISE_DS/ISE/coregen/ip/xilinx/dsp/com/xilinx/ip/cordic_v4_0
SET_CORE_GUIPATH D:/Xilinx/14.6/ISE_DS/ISE/coregen/ip/xilinx/dsp/com/xilinx/ip/cordic_v4_0/gui/cordic_v4_0.tcl
SET_CORE_DATASHEET D:\Xilinx\14.6\ISE_DS\ISE\coregen\ip\xilinx\dsp\com\xilinx\ip\cordic_v4_0\doc\cordic_ds249.pdf
ADD_CORE_DOCUMENT <D:\Xilinx\14.6\ISE_DS\ISE\coregen\ip\xilinx\dsp\com\xilinx\ip\cordic_v4_0\doc\cordic_ds249.pdf><cordic_ds249.pdf>
ADD_CORE_DOCUMENT <D:\Xilinx\14.6\ISE_DS\ISE\coregen\ip\xilinx\dsp\com\xilinx\ip\cordic_v4_0\doc\cordic_v4_0_vinfo.html><cordic_v4_0_vinfo.html>
