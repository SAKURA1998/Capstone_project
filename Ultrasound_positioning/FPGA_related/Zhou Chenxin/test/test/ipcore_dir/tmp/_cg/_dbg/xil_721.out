SET_PARAMETER pipelining_mode No_Pipelining
SET_PARAMETER ce false
SET_PARAMETER x_out true
SET_PARAMETER iterations 0
SET_PARAMETER functional_selection Sin_and_Cos
SET_PARAMETER coarse_rotation true
SET_PARAMETER register_outputs true
SET_PARAMETER nd false
SET_PARAMETER phase_format Scaled_Radians
SET_PARAMETER y_out false
SET_PARAMETER round_mode Nearest_Even
SET_PARAMETER architectural_configuration Parallel
SET_PARAMETER data_format SignedFraction
SET_PARAMETER output_width 16
SET_PARAMETER component_name sin
SET_PARAMETER precision 0
SET_PARAMETER sclr false
SET_PARAMETER phase_output false
SET_PARAMETER register_inputs true
SET_PARAMETER input_width 16
SET_PARAMETER compensation_scaling No_Scale_Compensation
SET_PARAMETER rdy false
SET_ERROR_CODE 2
SET_ERROR_MSG  CANCEL: Customization cancelled.
SET_ERROR_TEXT Finished initializing IP model.	
